app/components/Events/Form.js

```diff
@@ -56,6 +58,17 @@ const EventForm = memo(
             onChange={(e) => setDescription(e.target.value)}
           />
         </Form.Group>
+        {slug && (
+          <Form.Group className="has-float-label mb-4 col-12">
+            <Form.Label>Slug</Form.Label>
+            <Form.Input
+              type="text"
+              name="slug"
+              value={slug}
+              onChange={(e) => setSlug(e.target.value)}
+            />
+          </Form.Group>
+        )}
```

Comment:
Should me rendered always for edit show. Have an extra prop `showSlug` and pass it as true from EditModal